Support document, variables and headers in AS Landing Page config (#5711)
apollographql/apollo-server: docs/source/api/plugin/landing-pages.md

</td>
</tr>

<tr>
<td>

###### `document`

`string`
</td>
<td>

> You can pass valid graphql in your config via `document`, and when folks go to your Sandbox, this graphql will prepopulate in the editor pane.

Author: ```suggestion
A URI-encoded operation to populate in the Studio Explorer's editor on load.

If you omit this, the Explorer initially loads an example query based on your schema.
```

</td>
</tr>

<tr>
<td>

###### `variables`

`Record<string, string>`
</td>
<td>

> You can pass variables as a JSON object in your config, and when folks go to your Sandbox, these variables will prepopulate in the variables section of the Sandbox.

Author: ```suggestion
A URI-encoded, serialized object containing initial variable values to populate in the Explorer on load.

If provided, these variables should apply to the initial query you provide for document.
```

</td>
</tr>

<tr>
<td>

###### `headers`

`Record<string, string>`
</td>
<td>

> You can pass headers as a JSON object in your config, and when folks go to your Sandbox, these headers will prepopulate in the headers section of the Sandbox.

Author: ```suggestion
A URI-encoded, serialized object containing initial HTTP header values to populate in the Explorer on load.

```

</td>
</tr>

<tr>
<td>

###### `document`

`string`
</td>
<td>

A URI-encoded operation to populate in the Studio Explorer's editor on load.

If you omit this, the Explorer initially loads an example query based on your schema.

</td>
</tr>

<tr>
<td>

###### `variables`

`Record<string, string>`
</td>
<td>

You can pass variables as a JSON object in your config, and when folks go to your Sandbox, these variables will prepopulate in the variables section of the Sandbox.

</td>
</tr>

<tr>
<td>

###### `headers`

`Record<string, string>`
</td>
<td>

> A URI-encoded, serialized object containing initial HTTP header values to populate in the Explorer on load.

Author: ```suggestion
An object containing initial HTTP header values to populate in the Explorer on load.
```

</td>
</tr>

<tr>
<td>

###### `document`

`string`
</td>
<td>

> A URI-encoded operation to populate in the Studio Explorer's editor on load.

Author: ```suggestion
A document to populate in the Studio Explorer's editor on load.
```

If you omit this, the Explorer initially loads an example query based on your schema.

</td>
</tr>

<tr>
<td>

###### `variables`

`Record<string, string>`
</td>
<td>

> A URI-encoded, serialized object containing initial variable values to populate in the Explorer on load.
> 
> If provided, these variables should apply to the initial query you provide for document.

Author: ```suggestion
An object containing initial variable values to populate in the Explorer on load.

If provided, these variables should apply to the initial query you provide for document.
```
